Sources/Private/Experimental/Layers/GradientRenderLayer.swift

```diff
@@ -59,7 +59,7 @@ final class GradientRenderLayer: CAGradientLayer {
   ///  - This specific value is arbitrary and can be increased if necessary.
   ///    Theoretically this should be "infinite", to match the behavior of
   ///    `CGContext.drawLinearGradient` with `[.drawsAfterEndLocation, .drawsBeforeStartLocation]`.
-  private let gradientPadding: CGFloat = 1_000
+  private let gradientPadding: CGFloat = 2_000
```

Comment:
I found a case where this padding was too small, so I bumped it higher.

| Before (the stroke and gradient don't completely overlap) | After |
| ----- | ----- |
| <img width="172" alt="Screen Shot 2022-02-10 at 2 03 07 PM" src="https://user-images.githubusercontent.com/1811727/153518297-44e72c23-0352-465a-aa73-07ab5b4ab6e6.png"> | <img width="239" alt="Screen Shot 2022-02-10 at 2 05 58 PM" src="https://user-images.githubusercontent.com/1811727/153518306-cddcd02c-7953-4cd5-bf63-9627a5ddac20.png"> |

Reply:
that layer having a bounds of like 10000 is probably why it performs so poorly in the main thread engine 😛